Makefile

```diff
@@ -4,6 +4,7 @@ all:
 	@echo "Project successfully compiled"
 
 run-server:
+	rm -rf mongodb
```

Comment:
May prefer moving this into a `clean` target instead of being part of the `run-server` target.

Reply:
Done